portfolio/src/main/webapp/index.html

```diff
@@ -16,6 +16,7 @@ <h1>Quinn!</h1>
       <button onclick="addFact()" id = "fact-button">Click for a Fact!</button>
       <img src = "Images/me.jpg" id = "pics" align="left"/>
       <p> I'm very interested in audio! I've implemented dozens of audio manipulation techniques in python including denoisers, sound seperators, classifiers, etc. I'm currently working on a scuffed real time sound seperator that'll help with my jazz transcriptions.</p>
+      <a href="https://github.com/QuinnDACollins"><img class = "widget" src = "Images/github.png"/></a> <a href = "https://www.linkedin.com/in/quinn-collins-28a46416a/"><img class = "widget" src = "Images/LinkedIn.png"/></a>
```

Comment:
Typically directory names should be lowercase. So Images/LinkedIn.png -> images/linkedin.png.